src/encodings.rs

```diff
@@ -97,6 +107,14 @@ impl Encoding for RawEncoding {
     fn encode(&self) -> &Vec<u8> {
         &self.pixels
     }
+
+    fn transform(&self, input: &PixelFormat, output: &PixelFormat) -> Box<dyn Encoding> {
+        // XXX: This assumes the pixel formats are both rgb888. The server code verifies this
```

Comment:
Worth just doing an assert?

Reply:
yeah, i do one further down the chain but i don't think it hurts to add another here for now.

done